tests/parametric/test_otel_logs.py

```diff
@@ -0,0 +1,631 @@
+    """FR06: OTLP Protocol Tests"""
+
+    @pytest.mark.parametrize(
+        "library_env",
+        [
+            {
+                "DD_LOGS_OTEL_ENABLED": "true",
+                "OTEL_EXPORTER_OTLP_PROTOCOL": "http/protobuf",
+                "DD_TRACE_DEBUG": None,
+            },
+            {
+                "DD_LOGS_OTEL_ENABLED": "true",
+                "OTEL_EXPORTER_OTLP_PROTOCOL": "grpc",
+                "DD_TRACE_DEBUG": None,
+            },
+        ],
+        ids=["http_protobuf", "grpc"],
+    )
+    def test_otlp_protocols(self, test_agent, test_library, library_env):
+        """OTLP logs are emitted in expected format."""
+        with test_library as library:
+            library.write_log("test_otlp_protocols", LogLevel.INFO, "test_logger")
+
+        log_payloads = test_agent.wait_for_num_log_payloads(1)
+        assert find_log_record(log_payloads, "test_logger", "test_otlp_protocols") is not None
+
+
+@features.otel_logs_enabled
+@scenarios.parametric
+class Test_FR07_Host_Name:
+    """FR07: Host Name Attribute Tests"""
+
+    @pytest.mark.parametrize(
+        "library_env",
+        [
+            {
+                "DD_LOGS_OTEL_ENABLED": "true",
+                "DD_HOSTNAME": "ddhostname",
+                "DD_TRACE_REPORT_HOSTNAME": "true",
+                "DD_TRACE_DEBUG": None,
+            },
+        ],
+    )
+    def test_hostname_from_dd_hostname(self, test_agent, test_library, library_env):
+        """host.name is set from DD_HOSTNAME."""
+        with test_library as library:
+            library.write_log("test_hostname_from_dd_hostname", LogLevel.INFO, "test_logger")
+
+        log_payloads = test_agent.wait_for_num_log_payloads(1)
+        resource = find_resource(log_payloads, "test_logger", "test_hostname_from_dd_hostname")
+        attrs = find_attributes(resource)
+
+        assert attrs.get("host.name") == "ddhostname"
+
+    @pytest.mark.parametrize(
+        "library_env",
+        [
+            {
+                "DD_LOGS_OTEL_ENABLED": "true",
+                "OTEL_RESOURCE_ATTRIBUTES": "host.name=otelenv-host",
+                "DD_HOSTNAME": "ddhostname",
+                "DD_TRACE_REPORT_HOSTNAME": "true",
+                "DD_TRACE_DEBUG": None,
+            },
+        ],
+    )
+    def test_hostname_from_otel_resources(self, test_agent, test_library, library_env):
+        """OTEL_RESOURCE_ATTRIBUTES host.name takes precedence over DD_HOSTNAME."""
+        with test_library as library:
+            library.write_log("test_hostname_from_otel_resources", LogLevel.INFO, "test_logger")
+
+        log_payloads = test_agent.wait_for_num_log_payloads(1)
+        resource = find_resource(log_payloads, "test_logger", "test_hostname_from_otel_resources")
+        attrs = find_attributes(resource)
+
+        assert attrs.get("host.name") == "otelenv-host"
+
+    @pytest.mark.parametrize(
+        "library_env",
+        [
+            {
+                "DD_LOGS_OTEL_ENABLED": "true",
```

Comment:
Can you please add `"DD_HOSTNAME": "ddhostname",` to both of these test cases, so the difference between enabled/disabled is very clearly just the `DD_TRACE_REPORT_HOSTNAME` config?